### PR TITLE
Upgrade to PureScript 0.12

### PR DESCRIPTION
**Pull request by paluh**

### CODE REVIEW

File: src/Main.purs [hunk 1]
```diff
@@ -15,15 +14,18 @@ import Data.NonEmpty ((:|))
...
 import Node.FS.Stats (isDirectory)
 import Node.Optlicative (logErrors, optlicate)
 import Node.Path (FilePath)
 import Opts (Options, cmdOpts, prefs)
-import Snail (Script, Snail, cat, crawl, exists, exitWith, file, folder, mkdir, run, (+>), (~?>))
+import Snail (cat, crawl, exists, exitWith, file, folder, mkdir, run, (+>), (~?>))
 import Transliterate (transliterate)
 
-type App = Snail ()
+type App = Aff
```
Comment (Thimoteus) [inline src/Main.purs]: We can probably get rid of this alias, especially since this is an application and not a library

Comment (paluh) [reply src/Main.purs]: No problem

File: package.json [hunk 2]
```diff
@@ -21,5 +21,8 @@
     "purescript"
   ],
   "author": "Thimoteus",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```
Comment (paluh) [inline package.json]: I'm not sure if we want to have any `dependencies` here. Should I drop this `pulp` from here?

Comment (Thimoteus) [reply package.json]: Yeah, if anything it should be a `devDependencies`.

**Comment by Thimoteus** [comment]
Is this ready?

**Comment by paluh** [comment]
I think so :-)

**Comment by Thimoteus** [comment]
Alright, looks good, thanks!

**Pull request merged**